### PR TITLE
Rearranged header with css grid, added news link

### PR DESCRIPTION
**Pull request by DamienBurkeWN**
### Testing
* `yarn test Header`
* Try out header component on various screen resolutions to see how it behaves on mobile, tablet, desktop in portrait and landscape orientations

### CODE REVIEW

File: src/library/structure/Header/Header.styles.js [hunk 1]
```diff
@@ -1,54 +1,59 @@
...
 import MaxWidthContainer from '../MaxWidthContainer/MaxWidthContainer';
 
 /**
  * See header.tsx for colour rules
- * @param {*} props 
- * @returns 
+ * @param {*} props
+ * @returns
  */
-const headerThemeStyles = props => {
```
Comment (DamienBurkeWN) [inline src/library/structure/Header/Header.styles.js:9]: Not deleted, just prettier, below

File: src/library/structure/Header/Header.styles.js [hunk 2]
```diff
@@ -1,54 +1,59 @@
...
-    -webkit-flex-direction: row;
-    -moz-flex-direction: row;
-    -ms-flex-direction: row;
-    flex-direction: row;
-    justify-content: space-between;
-    align-items: center;
-    flex-wrap: wrap;
-    overflow: visible;
-`
+  display: grid;
```
Comment (DamienBurkeWN) [inline src/library/structure/Header/Header.styles.js:42]: Changed to use 3 different grid layouts depending upon resolution

File: src/library/structure/Header/Header.styles.js [hunk 3]
```diff
@@ -64,113 +69,106 @@ const LogoStyles = `
...
 export const HomeLink = styled.a`
-    display: inline-block;
+  display: inline-block;
 
-    &:hover {
-        opacity: 0.8;
+  &:hover {
+    opacity: 0.8;
 
-        svg {
```
Comment (DamienBurkeWN) [inline src/library/structure/Header/Header.styles.js:88]: Not deleted just prettier again

File: src/library/structure/Header/Header.styles.js [hunk 4]
```diff
@@ -64,113 +69,106 @@ const LogoStyles = `
...
+  @media screen and (max-width: 800px) {
+    padding-right: 0;
+  }
+  @media screen and (max-width: 400px) {
+    padding-right: 0;
+    text-align: center;
+  }
+`;
 
-const headerServicesLinkStyles = props => {
```
Comment (DamienBurkeWN) [inline src/library/structure/Header/Header.styles.js:101]: Not deleted just prettier again, and renamed to commonLinkStyles (above)

File: src/library/structure/Header/Header.stories.tsx [hunk 5]
```diff
@@ -38,13 +41,23 @@ const Template: Story<HeaderProps> = (args) => <Header {...args}>Children of the
 
 export const HeaderExample = Template.bind({});
 HeaderExample.args = {
+  hasNewsLink: true,
```
[the remark this answers is not in the document]
Comment (DamienBurkeWN) [reply src/library/structure/Header/Header.stories.tsx:44]: Strange, checking...

Browsers - from previous discussion it's Edge, Chrome, Firefox, Safari on Windows, MacOS, iOS, Android and Samsung Internet on Android. So we're all good on that score.

**Review by chrisrhymes — APPROVED** [review]
👍

**Pull request merged**